docs/ref/models/fields.txt

```diff
@@ -1085,6 +1085,16 @@ It uses :class:`~django.core.validators.validate_slug` or
     If ``True``, the field accepts Unicode letters in addition to ASCII
     letters. Defaults to ``False``.
 
+``SmallAutoField``
+------------------
+
+.. class:: SmallAutoField(**options)
+
+.. versionadded:: 3.0
+
+A 16-bit integer, much like an :class:`AutoField` except that it is
+guaranteed to fit numbers from ``1`` to ``32767``.
```

Comment:
This is not true for all supported databases, maybe:
```
Like an :class:`AutoField`, but only allows values under a certain
(database-dependent) point. Values from ``1`` ``32767`` are safe in all
databases supported by Django.
```

Reply:
Yeah, I adapted this from `BigAutoField`. Can tweak.

Reply:
Will use `limit` rather than `point` as it reads better.